samples/CommunityToolkit.Maui.Sample/MauiProgram.cs

```diff
@@ -14,81 +20,162 @@ public static class MauiProgram
 	public static MauiApp CreateMauiApp()
 	{
 		var builder = MauiApp.CreateBuilder();
-		builder.UseMauiApp<App>().UseMauiCommunityToolkit();
+		builder.UseMauiApp<App>().UseMauiCommunityToolkit().UseMauiCommunityToolkitMarkup();
```

Comment:
Just FYI - I noticed we forgot to add `UseMauiCommunityToolkitMarkup()`, even though we are using `CommunityToolkit.Maui.Markup` in the sample app. 

So I decided it'd be easiest to add it to this PR rather an opening a new PR.